app/views/home/index.html.erb

```diff
@@ -1,2 +1,11 @@
-<h1 class="text-center">Khan Academy Myanmar Community</h1>
-<p>This is the home page.</p>
+<div class="row">
+	<div class="col-xl-12 col-lg-12 col-md-12 col-sm-12 col-xs-12" style="float:left;text-align:center;margin-top:100px;">
+    <img src='/khan-academy.png'>
+  </div>
+  <div class="col-x1-12 col-lg-12 col-md-12 col-sm-12 col-xs-12 " style="float:right;text-align:center;margin-top:-20px;">
```

Comment:
Note on line 2 regarding grid classes and HTML styling.

Hardcoding your `margin-top` is also kind of jank; you probably just want to crop the image.